.github/workflows/ci.yml

```diff
@@ -185,7 +186,7 @@ jobs:
           ./linux-container-ci.sh ${{ env.BUILDER_VERSION }} aws-crt-${{ env.LINUX_BASE_IMAGE }} build -p ${{ env.PACKAGE_NAME }} --compiler=${{ matrix.compiler }}
 
   use-system-libcrypto:
-    runs-on: ubuntu-20.04 # latest
+    runs-on: ubuntu-22.04 # latest
```

Comment:
Why 22?

Reply:
24 is openssl3.0. 22 is still 1.1.1

Reply:
Maybe update the comment from # latest to explain the OpenSSL difference.

Reply:
actually, nvm. looks like 22 is now bundling openssl3.0 as well. lets switch to 24 then, c-cal still provides coverage for 1.1.1.